pages/integrations/sso.md.erb

```diff
@@ -89,3 +89,18 @@ No, SSO providers are setup using a unique identifier and are unaffected when a
 
 ### Can I merge two organizations that use different SSO providers?  
 In short, yes, you can. However, merging Buildkite organizations that already have SSO providers might be a tricky scenario, and it's highly recommended that you contact support@buildkite.com for help or guidance before you attempt such migration.  
+
+### Why am I being asked for my password in the "Authorization Required" screen on top of signing in to Buildkite using my organization's SSO? Can this authorization process be simplified?  
```

Comment:
Can we try to shorten these titles? How about 

> ### Why am I being asked for my password in the "Authorization Required" screen when signing in using SSO?

Reply:
Done, thanks!